modules/ocf_irc/files/znc.service

```diff
@@ -0,0 +1,11 @@
+[Service]
+ExecStart=/usr/bin/znc --foreground --datadir /opt/znc
+User=znc
+Group=znc
+Restart=always
+StandardOutput=syslog
+StandardError=syslog
+SyslogIdentifier=znc
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
Maybe include the reload command here like the one from the [docs](https://www.ocf.berkeley.edu/docs/staff/staffvm/znc/)? I don't think the syslog or stdout/stderr stuff is necessary either probably, but I also haven't tested it.